backend/src/routes/get.ts

```diff
@@ -0,0 +1,32 @@
+import { Elysia } from 'elysia';
+
+import { sql } from '@/lib/db';
+
+export const getRoute = new Elysia({ prefix: '/get' }).get(
+    '/email',
+    async ({ cookie }) => {
+        const sessionID = cookie.auth_session.value;
+
+        try {
+            const email = await sql`
+        SELECT auth_user.email
+        FROM auth_user 
+        INNER JOIN user_session
+        ON user_session.user_id=auth_user.id
+        WHERE user_session.id=${sessionID};
+      `;
+
+            return {
+                status: 'success',
+                data: {
+                    email: email,
+                },
+            };
+        } catch (error) {
+            return {
+                status: 'error',
+                message: 'Error occured somewhere, please try again',
+            };
+        }
```

Comment:
Does this route still in use? after fixing where required data is from /sign-in instead? Please remove this route if no longer in use

Reply:
removed